runhouse/resources/hardware/cluster.py

```diff
@@ -551,11 +551,24 @@ def _sync_default_env_to_cluster(self):
             logger.info("Disabling observability on the cluster")
 
         logger.info(f"Syncing default env {self._default_env.name} to cluster")
+
+        from runhouse.resources.envs.utils import _process_env_vars, run_setup_command
```

Comment:
do we want to move these somewhere else out of env.utils?

Reply:
yeah maybe a separate PR moving this out and can stack on that?